packages/ListBox/src/hooks/hooks.js

```diff
@@ -110,56 +110,6 @@ export function useAdjustWidth() {
   }, [dispatch, state.refTriggerContainer]);
 }
 
-export function useOnScrolled() {
```

Comment:
this was not longer relevant after we move the key up key down behaviour to focus on the next element instead of moving the scroll manually.